### DISCUSSION TITLE
production supervisor not firing handle

### DISCUSSION THREAD

**Discussion by Silassiai** [laravel/framework#35462] [Q&A] [answered]
- Laravel Version: 7.0
- PHP Version: 7.4
- Database Driver & Version: postgresql
- supervisor version: (3.3.1-1.1 (newest version)
- APP_ENV=Development (digital ocean ubuntu 18.04)

### Description:
Hi,

I am getting this error found in the supervisorlog. The job is called but never reaches the handle method.
However on my local server there is no problem. And when i run the artisan queue:work from the commandline in the development server it is also working. 

i did a var_dump on the var_dump($command); 
look at results like var_dump(__PHP_Incomplete_Class)

### Steps To Reproduce:

 FakeSubscribers::dispatch($from, $to)

`[2020-12-02 17:35:24][16] Processing: App\Jobs\FakeSubscribers
object(__PHP_Incomplete_Class)#1243 (11) {
  ["__PHP_Incomplete_Class_Name"]=>
  string(24) "App\Jobs\FakeSubscribers"
  ["from"]=>
  string(1) "4"
  ["to"]=>
  string(3) "100"
  ["job"]=>
  NULL
  ["connection"]=>
  string(5) "redis"
  ["queue"]=>
  string(5) "seeds"
  ["chainConnection"]=>
  NULL
  ["chainQueue"]=>
  NULL
  ["delay"]=>
  NULL
  ["middleware"]=>
  array(0) {
  }
  ["chained"]=>
  array(0) {
  }
}

   Symfony\Component\ErrorHandler\Error\FatalError

  method_exists(): The script tried to execute a method or access a property of an incomplete object. Please ensure that the class definition "App\Jobs\FakeSubscribers" of the object you are trying to operate on was loaded _before_ unserialize() gets called or provide an autoloader to load the class definition

  at /var/www/development.appdi.rect.to/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php:81
    77|     protected function dispatchThroughMiddleware(Job $job, $command)
    78|     {
    79| var_dump($command);
    80|         return (new Pipeline($this->container))->send($command)
  > 81|                 ->through(array_merge(method_exists($command, 'middleware') ? $command->middleware() : [], $command->middleware ?? []))
    82|                 ->then(function ($command) use ($job) {
    83|                     return $this->dispatcher->dispatchNow(
    84|                         $command, $this->resolveHandler($job, $command)
    85|                     );

   Whoops\Exception\ErrorException

  method_exists(): The script tried to execute a method or access a property of an incomplete object. Please ensure that the class definition "App\Jobs\FakeSubscribers" of the object you are trying to operate on was loaded _before_ unserialize() gets called or provide an autoloader to load the class definition

  at /var/www/development.appdi.rect.to/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php:81
    77|     protected function dispatchThroughMiddleware(Job $job, $command)
    78|     {
    79| var_dump($command);
    80|         return (new Pipeline($this->container))->send($command)
  > 81|                 ->through(array_merge(method_exists($command, 'middleware') ? $command->middleware() : [], $command->middleware ?? []))
    82|                 ->then(function ($command) use ($job) {
    83|                     return $this->dispatcher->dispatchNow(
    84|                         $command, $this->resolveHandler($job, $command)
    85|                     );

      +1 vendor frames
  2   [internal]:0
      Whoops\Run::handleShutdown()

`

**Comment by Silassiai** [comment, marked as answer]
After hours of debugging i finally found the problem.
I forgot that i changed the domain so my supervisor was running the old domain.

**Reply by lotje-kinable** [reply]
thank you!!!!!!
